CMakeLists.txt

```diff
@@ -179,6 +179,15 @@ install(
   PATTERN "*/test*" EXCLUDE
 )
 
+# Install CCF Python tools
+install(
+  DIRECTORY python/ccf
+  DESTINATION bin/
+  FILES_MATCHING
+  PATTERN "*.py"
+  PATTERN "*/__pycache__*" EXCLUDE
+)
+
 # Install CCF Python infrastructure
```

Comment:
We're still installing `infra/`, and the helper scripts, and `start_network.py`. We should work out which of these can be removed, though probably in a future PR once this is on PyPI.

Reply:
Absolutely. Added a task for this under #1382 for when automated releases are done.